dstep/driver/Application.d

```diff
@@ -34,30 +38,40 @@ class Application : DStack.Application
 
     private
     {
-        string[] inputFiles;
-
-        Index index;
-        TranslationUnit translationUnit;
-        DiagnosticVisitor diagnostics;
-
+        bool helpFlag;
         Language language;
         string[] argsToRestore;
-        bool helpFlag;
         Compiler compiler;
     }
 
     protected override void run ()
     {
         handleArguments;
-
         if (arguments.argument.input.hasValue)
         {
-            inputFiles ~= arguments.argument.input;
-            startConversion(inputFiles.first);
+            int j = 0;
+            auto remArgs = remainingArgs;
+            if(remArgs.length > 0)
+            foreach(string fileName; remArgs)
```

Comment:
I used parallel foreach in the same code, but for some reason it didn't work! That's why changed it into tasks... I'll try again though.